app/src/main/java/com/aileyzhang/musicapp/views/SongListItemView.java

```diff
@@ -0,0 +1,26 @@
+package com.aileyzhang.musicapp.views;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.view.View;
+import android.widget.FrameLayout;
+
+import com.aileyzhang.musicapp.R;
+
+/**
+ * Created by Ailey on 2017-12-18.
+ */
+
+public class SongListItemView extends FrameLayout {
+    View view;
+
+    public SongListItemView(@NonNull Context context) {
+        super(context);
+        initView();
+    }
+
+    private void initView() {
+        view = inflate(getContext(), R.layout.song_list_item, null);
```

Comment:
Could you see if `inflate(getContext(), R.layout.song_list_item, this);` and remove the `addView` line works? That last param should be the "parent" that you wanna inflate the view into - I think essentially doing these two lines in just one call. Lemme know if that works ~ other than that everything else looks great c: good workk